conf/defaults.yaml

```diff
@@ -251,10 +251,14 @@ topology.disruptor.batch.timeout.millis: 1
 topology.disable.loadaware: false
 
 # Configs for Resource Aware Scheduler
+topology.priority: 29
```

Comment:
Maybe add a comment here describing why 29, and what ranges can users use (Min, Max).